Jetcaster/app/src/main/java/com/example/jetcaster/data/PodcastsRepository.kt

```diff
@@ -43,8 +43,9 @@ class PodcastsRepository(
             refreshingJob?.join()
         } else if (force || podcastStore.isEmpty()) {
             refreshingJob = scope.launch {
+                try {
                 // Now fetch the podcasts, and add each to each store
-                podcastsFetcher(SampleFeeds).collect { (podcast, episodes, categories) ->
+                podcastsFetcher(SampleFeeds).collect{  (podcast, episodes, categories) ->
```

Comment:
Run the code formatting tool here. This line needs more indentation and a space should go between `collect` and the square bracket